develop/index.md

```diff
@@ -116,12 +116,14 @@ Integrate with the Polkadot blockchain's underlying protocol features to create
     <hr>
     - [Install and Get Started with Polkadot.js API](/develop/toolkit/api-libraries/polkadot-js-api/){target=\_blank}
     - [Install and Get Started with Polkadot-API (PAPI)](/develop/toolkit/api-libraries/papi/){target=\_blank}
+    - [Install and Get Started with Dedot](/develop/toolkit/api-libraries/dedot/){target=\_blank}
```

Comment:
Perhaps list in alphabetical order as to now show any biases to other APIs.

Reply:
Great idea, would love this as well. But I'm not sure what's the current list ordering criteria from the maintainers of the repo, it'd be great to some feedback here from the maintainers, I can adjust the position as needed.